RocketChat/Rocket.Chat
# [FIX] Description text error and icon missing Chatpal

## Issue(s)

The description of some settings has been parsed wrong, hence the messages are now written in Markdown and not HTML. Also the icons by search results In Chatpal plug-in didn't appear. 

## Steps to test or reproduce

1. Create a channel and set up Chatpal search
2. Write some test messages
3. Try to search for them with the search plugin
4. Next to your username should appear your avatar
